css/onoffice_defaultview.css

```diff
@@ -235,7 +235,15 @@
 }
 
 .oo-asplocation {
-	margin-bottom: 10px;
+	margin-bottom: 15px;
```

Comment:
This change would affect all running WP instances with this plugin, because `onoffice_defaultview.css` is always included. This change can break the style of existing sites.